commands/command_smudge.go

```diff
@@ -20,6 +22,57 @@ var (
 	smudgeSkip = false
 )
 
+// delayedSmudge performs a 'delayed' smudge, adding the LFS pointer to the
+// `*tq.TransferQueue` "tq" if the file is not present locally, passes the given
+// filepathfilter, and is not skipped. If the pointer is malformed, or already
+// exists, it streams the contents to be written into the working copy to "to".
+//
+// delayedSmudge returns the number of bytes written, whether the checkout was
+// delayed, the *lfs.Pointer that was smudged, and an error, if one occurred.
+func delayedSmudge(s *git.FilterProcessScanner, to io.Writer, from io.Reader, tq *tq.TransferQueue, filename string, skip bool, filter *filepathfilter.Filter) (int64, bool, *lfs.Pointer, error) {
```

Comment:
Good call -- I think we should be fine for now, since we don't use any other exported members from the `tq` package, but this will be a good way to prevent an ambiguous diff later on if that changes. Covered this one in: a9942cd.